fiftyone/utils/eval/coco.py

```diff
@@ -134,16 +330,36 @@ def _coco_evaluation(gts, preds, eval_key, config):
         preds = objects["preds"]
 
         # Highest confidence predictions first
-        preds = sorted(preds, key=lambda p: p.confidence or -1, reverse=True)
+        preds = sorted(preds, key=lambda p: p.confidence or -1, reverse=True)[
```

Comment:
Although COCO may limit to 100 predictions, perhaps we don't want to do this? Or, at least, we should make this a parameter that the user can configure.

Certainly for the non-mAP part of this calculation, I think we should never cap the number of the predictions in this list, because the typical user of `evaluate_detections()` will expect that all of their predictions are evaluated, certainly for TP/FP/FN counts, for example.

Reply:
Perhaps the best approach is to convert this value `100` to a config parameter that can be set to `None` by the user if they do not want any limiting to be done.

Or, if you would rather think of IoU sweeps as *exactly* following COCO style, then leave it hard-coded to 100 but *only* for the IoU sweep portion of the evaluation.

Another COCO-specific parameter is `iscrowd`, which the evaluation routine provides support for customizing. And, importantly, evaluation will work if the user's data doesn't have a crowd parameter populated.